content/en/real_user_monitoring/guide/monitor-hybrid-react-native-applications.md

```diff
@@ -0,0 +1,327 @@
+---
+title: Monitor Hybrid React Native Applications
+kind: guide
+description: Guide for monitoring hybrid React Native applications.
+further_reading:
+- link: '/monitors/create/types/real_user_monitoring/'
+  tag: 'Documentation'
+  text: 'Learn about RUM Monitors'
+---
+
+## Overview
+
+If you have a hybrid application built on React Native, you can use Datadog to monitor the same application both from the React Native and native sides.
+
+RUM Events are reported as coming from the same application and the same source in Datadog RUM.
+
+## Limitations
+
+- For **errors, resources, and interactions tracking**, the SDKs can work in two ways:
+  - Through *auto-instrumentation* - Some React classes and methods are modified to automate this. Auto-instrumentation for JavaScript errors, resources, and interactions can only be started from JavaScript code.
+  - Through *manual instrumentation* - For example, if you want to report something you consider an error but is not going to crash the app. 
+- You can share the same instance of the core SDK between native and React Native. This allows you to initialize the native SDK on either the native side or on the React Native side (by calling `DdSdkReactNative.initialize`) and have it initialized for both sides, with events appearing in the same RUM session. React Native uses the default core instance. This means that you can use *manual instrumentation* on both sides, but *auto-instrumentation* is only be activated for the side that the SDK was initialized.
+- You can only report Datadog RUM events or legs after the initialization, otherwise they are not sent.
+- You cannot change the source attribute of a RUM session - all your RUM events appear under the same source.
+
+## Monitor React Native apps with native content
+
+Before you can start monitoring Reactive Native apps with native content, you need to initialize the React Native SDK.
+
+### Initialize the React Native SDK
+
+To initialize the React Native Datadog SDK for RUM, see the [React Native Monitoring documentation][1].
+
+This also initializes the SDK on the native side. You are able to call both the native and React Native SDKs for logs, traces and RUM.
+
+If you are sure that you don't call the native SDK before the React Native SDK this is the solution we recommend.
```

Comment:
@marco-saia-datadog The way this is phrased confuses me. Can you make sense of it?

Reply:
Yes, it is a bit confusing. I will rephrase it.